src/valkey-benchmark.c

```diff
@@ -1460,6 +1407,11 @@ int parseOptions(int argc, char **argv) {
                 config.num_threads = 0;
         } else if (!strcmp(argv[i], "--cluster")) {
             config.cluster_mode = 1;
+        } else if (!strcmp(argv[i], "--rfr")) {
+            config.read_from_replica = FROM_REPLICA_ONLY;
+            if (argv[i + 1] && atoi(argv[++i]) == 2) {
+                config.read_from_replica = FROM_ALL;
+            }
```

Comment:
Not sure if we should not enable accepting all possible values here. I agree it might be strange to configure rfr as 'no' or 'primaries' but it makes more sense to disable it but still provide the configuration. 
We can alternatively allow integer mode values but we should document them well in the help section.

Also this way the command limne is very error prone. it is better to always request a value tmatching to the --rfr and validate it is in the correct format (in case of integer check the range and in case string validate is matches the allowed values)